eng/performance/benchmark_jobs.yml

```diff
@@ -90,16 +90,19 @@ jobs:
         container: ${{ parameters.container }}
         strategy:
           matrix:
-            ${{ each framework in parameters.frameworks }}:
-              ${{ framework }}:
-                _Framework: ${{ framework }}
+            ${{ each channel in parameters.channels }}:
+              ${{ channel }}:
+                _Channel: ${{ channel }}
                 _Configs: CompilationMode=Tiered RunKind="${{ parameters.kind }}"
                 _PerfLabArguments: $(PerfLabArguments)
-                _BuildConfig: ${{ parameters.architecture }}_$(_Framework)_${{ parameters.kind }} # needs to be unique to avoid logs overwriting in mc.dot.net
+                _BuildConfig: ${{ parameters.architecture }}_$(_Channel)_${{ parameters.kind }} # needs to be unique to avoid logs overwriting in mc.dot.net
         steps:
         - checkout: self
           clean: true
-        - script: $(Python) scripts/ci_setup.py --frameworks $(_Framework) --architecture ${{parameters.architecture}} --perf-hash $(Build.SourceVersion) --queue ${{parameters.queue}} --build-number $(Build.BuildNumber) --build-configs $(_Configs)
+        - script: $(Python) scripts/channel_map.py --channel $(_Channel) --setup-pipeline # find the channel's matching tfm 
+          displayName: Look up TFM
+        - script: $(Python) scripts/ci_setup.py --channels $(_Channel) --architecture ${{parameters.architecture}} --perf-hash $(Build.SourceVersion) --queue ${{parameters.queue}} --build-number $(Build.BuildNumber) --build-configs $(_Configs)
```

Comment:
just curious: why did you decide to keep the logic from `channel_map.py` in a separate file? as far I understand it translates the channel to a tfm and then stores the result in some AZDO specific way (so if I run it locally it won't work?)? would it be better to just have it as part of `ci_setup.py`?

Reply:
Currently ```channel_map.py``` can be used by other scripts such as ```benchmarks_ci.py```, ```micro_benchmarks.py``` and ```ci_setup.py```, to get the right tfm for a channel; it is also used independently in the pipeline. Because we'd like to support local benchmark runs triggered by ```benchmarks_ci.py```, putting them in ```ci_setup.py``` would make it only available for ci runs. What do you think about this?

Reply:
New plan based on offline discussion: moving the content of ```channel_map.py``` to ```ci_setup.py``` sounds right, because 1) it's high-coupling for ```channel_map.py``` to know anything about the pipeline 2) if there's no need to run ```channel_map.py```, we can totally move the ```ChannelMap``` class to ```ci_setup.py``` and let ```benchmarks_ci.py``` use this module in local runs